Decred Wallet/view_controller/AccountViewController.swift

```diff
@@ -71,25 +71,17 @@ class AccountViewController: UIViewController, UITableViewDataSource, UITableVie
         
         DispatchQueue.global(qos: .background).async { [weak self] in
             guard let this = self else { return }
-            this.account?.Acc.removeAll()
             this.accountHeaders.removeAll()
-            do {
-                var getAccountError: NSError?
-                let strAccount = AppDelegate.walletLoader.wallet?.getAccounts(0, error: &getAccountError)
-                if getAccountError != nil {
-                    throw getAccountError!
-                }
-                
-                this.account = try JSONDecoder().decode(WalletAccounts.self, from: (strAccount?.data(using: .utf8))!)
+            
+            if let acc = AppDelegate.walletLoader.wallet?.walletAccounts(confirmations: 0) {
+                this.accounts = acc
                 this.accountHeaders = {
                     var colorCount = -1
-                    return this.account!.Acc.map {
+                    return acc.map {
                         colorCount += 1
                         return AccountHeader(entity: $0, color: nil)
                     }
                 }()
```

Comment:
Please simplify this to:
`this.accountHeaders = this.accounts.map({ AccountHeader(entity: $0, color: nil) })`

It appears the `colorCount` variable that was declared and incremented has no use.